src/pages/Calculator.tsx

```diff
@@ -174,26 +174,20 @@ export const Calculator = (): React.ReactElement => {
     </button>
   )
 
+  const oneWeekOfMilliseconds = 7 * 24 * 60 * 60 * 1000
```

Comment:
Could we use moment.js to do math on time instead of converting to milliseconds?

Reply:
I ended up with [date-fns](https://date-fns.org/); sounds like that's one of the things the [cool kids](https://dockyard.com/blog/2020/02/14/you-probably-don-t-need-moment-js-anymore) are using now that moment.js is [deprecated](https://momentjs.com/docs/#/-project-status/).  date-fns does seem to have nice immutable functions nad play nicely with tree shaking, but I don't have strong feelings about it if you'd prefer moment.js.

I retested using the same method.

Reply:
Looks good to me, thanks!

The hot library of the day moves so fast!